erpnext/accounts/report/accounts_receivable/accounts_receivable.py

```diff
@@ -88,6 +89,59 @@ def get_data(self):
 
 		self.build_data()
 
+		if self.filters.get("group_by_partner"):
+			self.data = sorted(self.data, key=lambda i: i['sales_partner'])
+			self.sales_partner_summary_lines()
+
+
+	def sales_partner_summary_lines(self):
+		"""
+		Insert summary lines for each sales partner if group by partner feature is requested.
+		"""
+		previous_partner = ""
+		outstanding_amount, paid_amount, invoiced_amount = 0.0, 0.0, 0.0
+		range1, range2, range3, range4 = 0,0,0,0
+		index = 0
+		data = []
+		for row in self.data:
+			index += 1
+			current_partner = row.get("sales_partner")
+
+			#create a summary line if the sales partner has changed, reset totals
+			if current_partner != previous_partner:
+				total = range1 + range2 + range3 + range4
+				partner_sales = "Unassigned" if previous_partner == "" else previous_partner
+				summary_line = {
+					"sales_partner": "Total for {0}".format(partner_sales),
+					"outstanding": outstanding_amount,
+					"paid": paid_amount,
+					"invoiced": invoiced_amount,
+					"range1": range1,
+					"range2": range2,
+					"range3": range3,
+					"range4": range4,
+					"total": total
+				}
+				data.append(summary_line)
+				data.append({})
```

Comment:
```suggestion
```

Reply:
We need an empty line after the summary line, hence I added this.